nodejs_container/README.md

```diff
@@ -46,8 +46,8 @@ const scenario = new Scenario([instanceAlice, instanceBob])
 //   - the second argument is an object containing an interface into each instance specified in the config
 scenario.runTape("test something", (t, {alice, bob}) => {
     // fire zome function calls from both agents
-    const result1 = alice.call('zome', 'capability', 'function', {params: 'go here'})
-    const result2 = bob.call('zome', 'capability', 'function', {params: 'go here'})
+    const result1 = alice.call('zome', 'function', {params: 'go here'})
+    const result2 = bob.call('zome', 'function', {params: 'go here'})
```

Comment:
Don't we still need the trait here?

Reply:
nope, no trait ever needed for calls, only the capability token which will be in the params.